test/integration/http2_integration_test.cc

```diff
@@ -1560,8 +1563,6 @@ void Http2FloodMitigationTest::floodServer(const Http2Frame& frame, const std::s
 
   EXPECT_LE(total_bytes_sent, TransmitThreshold) << "Flood mitigation is broken.";
   EXPECT_EQ(1, test_server_->counter(flood_stat)->value());
-  EXPECT_EQ(1,
```

Comment:
are we no longer incrementing these stats?

Reply:
Whoops, I meant to tag @yanavlasov as reviewer on this one and must have misclicked - good to have a non-googler anyway.

Yeah, the default is now to not wait 1s to drain, at which point the drain stat doesn't get incremented.  I don't think there's a need to drain close here (unless the tests flake on one of the CIs) so I'd rather avoid the 1s wait than wait and increment.